test/test_onnx.py

```diff
@@ -76,20 +76,6 @@ def to_numpy(tensor):
                 else:
                     raise
 
-    @unittest.skip("Disable test until Split w/ zero sizes is implemented in ORT")
-    def test_new_empty_tensor(self):
```

Comment:
My initial reaction was to keep this test, as it's not using the `new_empty_tensor` op but instead tries to enforce that `nn.Conv2d` support empty batch sizes (which was the original use-case for having a `new_empty_tensor`)

But given that this test has been disabled for a long time, let's just remove it.